Libraries/src/Amazon.Lambda.AspNetCoreServer/APIGatewayProxyFunction.cs

```diff
@@ -37,6 +39,16 @@ public abstract class APIGatewayProxyFunction
         public bool EnableResponseLogging { get; set; }
 
         /// <summary>
+        /// If true then all response content will be Base64-encoded before being returned.
+        /// </summary>
+        public bool EnableBinaryResponseContent { get; set; }
+
+        /// <summary>
+        /// An optional set of Content-Types that will trigger encoding the response content in Base64.
+        /// </summary>
+        public IEnumerable<string> BinaryResponseContentTypes { get; set; }
```

Comment:
What if we switch this around. Maintain a collection of common content types that we don't need to be base 64 encoded like text/html. Don't expose the data collection but add an AddTextContentType operation. Then when processing the response if the content type is not in the collection then base 64 encode it. 

This way we can make it so that must users don't have to anything special to handle binary type.